File: Chronos.App/DataContracts/Blank/BlankTask.cs
namespace Chronos.App.DataContracts.Blank
{
    public class BlankTask
    {
        public int? ParentStoryId { get; set; }
        
        public string Name { get; set; }

        public decimal OriginalEstimate { get; set; }

        public string Activity { get; set; }
    }
}

File: Chronos.App/ClientApp/src/app/models/blank/blank-task.ts
import { User } from "src/app/service";

export class BlankTask {
    parentStoryId: 0 ;
    name = '';
    originalEstimate = 0;
    assigned: User = null;
    activity = '';

    constructor(obj?: Partial<BlankTask>) {
        if (obj)
            Object.assign(this, obj);
    }
}
